backend/app/services/channels/telegram/service.py

```
@@ -0,0 +1,652 @@
+                # Show keyboard if enabled, otherwise show status
+                if self._handler._use_inline_keyboard:
+                    await self._handler.send_mode_keyboard(message_context, user)
+                else:
+                    await self._handler._handle_status_command(
+                        db, user, message_context
+                    )
+        finally:
+            db.close()
+
+    async def _handle_callback_query(
+        self, update: Update, context: ContextTypes.DEFAULT_TYPE
```

Ruff (0.14.14)

[warning] Unused method argument: `context`

(ARG002)